app/code/Magento/Ui/view/base/web/js/form/element/image-uploader.js

```diff
@@ -78,6 +78,7 @@ define([
                 '/type/image/?isAjax=true';
 
             if (this.mediaGallery.initialOpenSubpath) {
+                openDialogUrl += '/tree_path/' + this.mediaGallery.initialOpenSubpath;
```

Comment:
If we remove the parameter from the DefaultConfigProvider this won't be necessary, right?